back/init.sql

```diff
@@ -1592,15 +1600,23 @@ CREATE TABLE `shipment_detail` (
   KEY `shipment_detail_source_location_id` (`source_location_id`),
   KEY `shipment_detail_target_location_id` (`target_location_id`),
   KEY `shipment_detail_created_by_id` (`created_by_id`),
-  KEY `shipment_detail_deleted_by_id` (`deleted_by_id`),
+  KEY `shipment_detail_deleted_by_id` (`removed_by_id`),
```

Comment:
The naming is not matching anymore here, but I don't think it's an issue @HaGuesto